cpp/src/utility/z_order.cuh

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+// adapted from https://dl.acm.org/doi/10.1109/TC.2007.70814
+__constant__ const uint16_t dilate_tab2[256] = {
```

Comment:
Is it possible we can initialize this using `constexpr`? It's already written out, so I guess it doesn't matter, but it's difficult to verify it's correctness. :man_shrugging: :

Reply:
out of scope.